packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -172,13 +172,13 @@ contract PluginSetupProcessor is DaoAuthorizable {
 
     /// @notice Thrown when user's arguments for the apply function don't match the currently applied setupId.
     /// @param currentSetupId The current setup id to which user's preparation setup should match to.
```

Comment:
Which of the prepared/applied does this correspond to?

Reply:
every applied setupId.